.tekton/pipeline.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  creationTimestamp: '2023-12-03T20:32:31Z'
+  generation: 2
+  managedFields:
+    - apiVersion: tekton.dev/v1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          .: {}
+          'f:finally': {}
+          'f:params': {}
+          'f:tasks': {}
+          'f:workspaces': {}
+      manager: Mozilla
+      operation: Update
+      time: '2023-12-03T21:55:43Z'
+  name: cd-pipeline
+  namespace: luoyantao99-dev
```

Comment:
create a new namespace named "devops-produts", then all resource use this namespace

Reply:
I add it